docker-compose.yml

```diff
@@ -22,6 +22,11 @@ services:
       # Mount the development directory as a volume into the container, so
       # Docker automatically recognizes your changes.
       - .:/app
+    environment:
+      - IMPORT_DIRECTORY=${IMPORT_DIRECTORY}
+      - PG_HOST=${PG_HOST}
+      - PG_USER=${PG_USER}
+      - PG_PASSWORD=${PG_PASSWORD}
```

Comment:
This injects values from a specified env file from the build context into the container, allowing us to run commands like `docker-compose --env-file .env.import run --rm app make -e import_docket_import` with the correct values for local development.